src/components/AvMedia.js

```diff
@@ -1,10 +1,11 @@
+import { h } from "vue"
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/staskobzar/vue-audio-visual/pullRequest?prid=7811601)
```suggestion
import { h } from 'vue'
```